### PR TITLE
lineToVec display

### PR DESCRIPTION
**Pull request by yCobanoglu**
new lineToVec endpoint
and simple lineToVec display component

### CODE REVIEW

File: src/fragmentarium/infrastructure/FragmentRepository.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ import { FolioPagerData, FragmentPagerData } from 'fragmentarium/domain/pager'
 import { museumNumberToString } from 'fragmentarium/domain/MuseumNumber'
 import { Genres } from 'fragmentarium/domain/Genres'
 import Word from 'dictionary/domain/Word'
+import { LineToVecRanking } from '../domain/lineToVecRanking'
```
Comment (jlaasonen) [inline src/fragmentarium/infrastructure/FragmentRepository.ts:52]: Use absolute paths.

File: src/fragmentarium/ui/line-to-vec/FragmentLineToVecRanking.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  const RankingList = ({ score }) => {
+    return score.map((score, index) => (
+      <li key={index}>
+        {score[0]}:{score[1]}
+      </li>
+    ))
+  }
+  const LineToVecDisplay = ({ lineToVecRanking }) => (
+    <Container>
+      <Col>
```
Comment (jlaasonen) [inline src/fragmentarium/ui/line-to-vec/FragmentLineToVecRanking.tsx:27]: Remove extraneous element.

File: .env [hunk 3]
```diff
@@ -1 +0,0 @@
-INLINE_RUNTIME_CHUNK=false
```
Comment (jlaasonen) [inline .env:1]: Do not remove this.

**Review by jlaasonen — CHANGES_REQUESTED** [review]

**Comment by yCobanoglu** [comment]
[Count number of rulings in tablet to find matching sequences](https://trello.com/c/GRpAtWIj/47-count-number-of-rulings-in-tablet-to-find-matching-sequences)

**Review by jlaasonen — APPROVED** [review]

**Pull request merged**